docs/Developers/Development_Setup.rst

```diff
@@ -34,6 +34,24 @@ https://github.com/coala/coala-bears
 Installing from Git
 -------------------
 
+NOTE:
+In order to configure the environment according to python 3.
+If you have both python 3 and python 2 installed check this out first.
+
+::
+
+    $ virtualenv venv -p /usr/bin/python3
+
+Enter the path where you have installed python 3.
+
+::
+
+    $ source venv/bin/activate
+    (venv)$ python --version
+
+Check your python version and make sure it is python 3.
```

Comment:
@NiklasMM "I'd put this addition into the "virtualenv" section rather than here"
Can you please elaborate it?